src/main/java/nl/tudelft/cse1110/andy/execution/step/SourceCodeSecurityCheckStep.java

```diff
@@ -55,7 +55,9 @@ private boolean checkPackageName(String code, ResultBuilder result) {
 
     private boolean checkForKeywords(String code, ResultBuilder result) {
         String reflectionMsg = "Using reflection in your code is not allowed";
-        var keywords = Map.of(
+        String seleniumUnsupportedDriverMsg = "Please use HtmlUnitDriver as WebLab does not have a browser installed";
```

Comment:
Instead of "WebLab", shall we say that "Andy" doesn't support it? This way the message will look nice even in the command line version.